src/clientversion.cpp

```diff
@@ -99,5 +100,7 @@ std::string FormatSubVersion(const std::string& name, int nClientVersion, const
         ss << ")";
     }
     ss << "/";
+    if (!fBaseNameOnly && IsArgSet("-bip148"))
+        ss << "UASF-Segwit:0.3(BIP148)/";
```

Comment:
Good point.  Since there is zero difference in the code paths of the two clients, I think that distinction can only be confusing or even misleading.  As if one is "more official" than the other.